RUN3/AliAnalysisTaskAO2Dconverter.cxx

```diff
@@ -1239,6 +1239,10 @@ void AliAnalysisTaskAO2Dconverter::FillEventInTF()
     if (goldenChi2Status)
       tracks.fFlags |= TrackFlagsRun2Enum::GoldenChi2;
 
+    // Uppermost 4 bits contain PID hypothesis used during tracking
+    if (track->GetPIDForTracking() >= 0 && track->GetPIDForTracking() <= 15)
```

Comment:
in principle, the track always have a PID assigned (default pion), no need for `if`.

Reply:
I agree. The return type is int, therefore I wanted the protection to make sure the other bits are not touched.

Btw the 3 first bits are also free for run3 tracks : we have a separate enum for run2 and run3 (distinguishable by the track type)